index.html

```diff
@@ -0,0 +1,202 @@
+              <span></span>
+              <li><h1 class="year">2015</h1></li>
+            </ul>           
+        </div>
+        <h5 
+          class="bodyText">A daily selection of privately personalized reads; no accounts or sign-ups required.
+       </h5> 
+        <ul class="tag">
+          <li class="tags">
+            <h2 class="html">html</h2>
+          </li>
+          <li class="tags">
+            <h2 class="html">css</h2>
+          </li>
+          <li class="java">
+            <h2 class="html">javaScript</h2>
+          </li>
+        </ul>
+        <div id="action">
+          <button class="normalButtn"> See project</button>
+        </div>
+      </div>
+    </div>
+    <div class="card">
+      <div class="snapshot">
+        <img src="./asset/images/professional.svg" alt="" cl>
+      </div>
+      <div class="left block">
+        <div class="pryText">
+          <h1 class="tonicTxt">Tonic</h1>
+          <ul class="frame">
+            <li><h1 class="client">CANOPY</h1></li>
+            <span></span>
+            <li><h1 class="role">Back End Dev</h1></li>
+            <span></span>
+            <li><h1 class="year">2015</h1></li>
+          </ul>           
+        </div>
+        <h5 
+          class="bodyText">A daily selection of privately personalized reads; no accounts or sign-ups required.
+       </h5> 
+        <ul class="tag">
+          <li class="tags">
+            <h2 class="html">html</h2>
+          </li>
+          <li class="tags">
+            <h2 class="html">css</h2>
+          </li>
+          <li class="java">
+            <h2 class="html">javaScript</h2>
+          </li>
+        </ul>
+        <div id="action">
+          <button class="normalButtn"> See project</button>
+        </div>
+      </div>
+    </div>
+    <div class="cardMutiple">
+      <div class="snapshot">
+        <img src="./asset/images/yoga.svg" alt="" cl>
+      </div>
+      <div class="multipleLeft block">
+        <div class="multiplePryText">
+          <h1 class="multipleTxt">Multi-Post
+            Stories
+          </h1>
+          <ul class="frame">
+            <li><h1 class="client">CANOPY</h1></li>
+            <span></span>
+            <li><h1 class="role">Back End Dev</h1></li>
+            <span></span>
+            <li><h1 class="year">2015</h1></li>
+          </ul>            
+        </div>
+        <h5 
+          class="bodyText">A daily selection of privately personalized reads; no accounts or sign-ups required.
+       </h5> 
+       <ul class="tag">
+          <li class="tags">
+          <h2 class="html">html</h2>
+          </li>
+          <li class="tags">
+            <h2 class="html">css</h2>
+          </li>
+          <li class="java">
+            <h2 class="html">javaScript</h2>
+          </li>
+        </ul>
+        <div id="action">
+          <button class="normalButtn"> See project</button>
+        </div>
+      </div>
+    </div>
+  </section>  
```

Comment:
Missing the About section of the profile as was required in this milestone.
![image](https://user-images.githubusercontent.com/61361037/219770055-3284169b-4426-4d78-b908-ca535e632307.png)

Reply:
ok, I Thought i was on a different branch @RaoAkif, I will include the snippet now.